meson.build

```diff
@@ -2,20 +2,20 @@ project(
     'gnome-software-plugin-apk',
     ['c'],
     version: '0.9.2',
-    meson_version: '>=0.51'
+    meson_version: '>=0.58'
 )
 
 gnome_software_dep = dependency('gnome-software', version: '>=40.0')
-plugin_install_dir = gnome_software_dep.get_pkgconfig_variable('plugindir')
+plugin_install_dir = gnome_software_dep.get_variable('plugindir')
 
 cargs = ['-DG_LOG_DOMAIN="GsPluginApk"', '-DI_KNOW_THE_GNOME_SOFTWARE_API_IS_SUBJECT_TO_CHANGE']
 
 apk_dep = dependency('apk-polkit-client-1')
-glib_dep = dependency('glib-2.0')
+glib_dep = dependency('glib-2.0') # For tests, minimum 2.60 dependency
```

Comment:
```suggestion
glib_dep = dependency('glib-2.0', version: '>=2.60')
```

Reply:
Thanks! That was quite stupid on my side :(